junifer/data/atlases.py

```diff
@@ -174,37 +172,14 @@ def load_atlas(
     path_only : bool, optional
         If True, the atlas image will not be loaded (default False).
 
-    Extra Parameters
```

Comment:
Why are we removing this?

Reply:
Should not be here, there are no extra parameters in `load_atlas`